keps/2941-DRA/README.md

```diff
@@ -55,7 +61,7 @@ tags, and then generate with `hack/update-toc.sh`.
   - [Using a CEL expression](#using-a-cel-expression)
   - [Defining DeviceClass mapping in ClusterQuota](#defining-deviceclass-mapping-in-clusterquota)
   - [Using ResourceFlavor for DeviceClass Mapping](#using-resourceflavor-for-deviceclass-mapping)
-  - [Using ConfigMap for DeviceClass Mapping](#using-configmap-for-deviceclass-mapping)
+  - [Creating a new CRD for device class mapping](#creating-a-new-crd-for-device-class-mapping)
```

Comment:
I don’t think this subtitle is correct: we aren’t sure CRD anymore

Reply:
It's the alternatives section. I attempted to capture the previous design here in the alternatives

Reply:
Ah I see that! That’s what I get for reviewing on mobile.